### PR TITLE
Discourse Community integration

### PR DESCRIPTION
**Pull request by CodeMouse92**
* Replace all Zendesk links with equivalent Discourse links.
* 'Mailspring Help' menu option now points to /docs on Discourse.
* Add 'Mailspring Community' to menu.
* Remove old FAQ (same as new Mailspring Help).

### CODE REVIEW

File: app/src/browser/application.ts [hunk 1]
```diff
@@ -397,19 +397,19 @@ export default class Application extends EventEmitter {
     });
 
     this.on('application:view-help', () => {
-      const helpUrl = 'http://support.getmailspring.com/hc/en-us';
+      const helpUrl = 'https://community.getmailspring.com/docs';
```
Comment (bengotow) [inline app/src/browser/application.ts:397]: I wonder if this should link to https://community.getmailspring.com/categories? It's nice that this docs page has the search bar on it, but it seems to sort the issues beneath the bar by recency which is a bit of a bummer.

File: app/src/browser/application.ts [hunk 2]
```diff
@@ -397,19 +397,19 @@ export default class Application extends EventEmitter {
...
       shell.openExternal(helpUrl);
     });
 
     this.on('application:view-getting-started', () => {
       const helpUrl =
-        'https://foundry376.zendesk.com/hc/en-us/sections/115000521592-Getting-Started';
+        'https://community.getmailspring.com/pub/quick-start-guide';
```
Comment (bengotow) [inline app/src/browser/application.ts:403]: This looks great!

**Comment by bengotow** [comment]
This looks good to me! Thanks for moving these over to the new community site. I think this will definitely make it easier for folks to keep the information up to date. Let's go ahead and merge this in the next release and I'll work on disabling / redirecting the existing content soon. 

(We may not be able to redirect the old content since a lot of the URLs go directly to zendesk, which is annoying... May need to wait for Google to pick up the new content so things like https://www.google.com/search?client=firefox-b-1-d&q=mailspring+snooze+emails still return helpful results!)

File: app/src/browser/application.ts (hunk 1 above)
Comment (CodeMouse92) [reply app/src/browser/application.ts:397]: I still think we should use Docs, as it filters by topics, whereas the main categories does not. Both will sort by recency.

**Comment by CodeMouse92** [comment]
@bengotow I just took care of the merge conflict on this one.

**Comment by CodeMouse92** [comment]
Just a reminder: this one will *need* to make it into the next release, lest people keep going to the dead-end that is Zendesk.

**Review by Phylu — APPROVED** [review]

**Comment by bengotow** [comment]
Thanks @CodeMouse92! Will be nice to align the app with the new discourse pages - I'll also update the website to include links to Discourse this morning!

**Pull request merged**